followed-accounts.css

```diff
@@ -0,0 +1,47 @@
+html {
+  background: var(--bg-color);
+  font-family: var(--rdp-font);
+}
+
+.followed-container {
+  flex: 1;
+  max-width: 600px;
+  width: 100%;
+  margin: 0 20px;
+  color: var(--rdp-text-color);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.imp-exp-btn {
+  margin-bottom: 20px;
+}
+
+#exportFollowedList,
+#importFollowedList {
+  cursor: pointer;
+}
+
+.cache-warning-msg {
+  font-size: 0.775em;
+  margin-top: 10px;
+}
+
+followed-actors-list {
+  text-align: left;
+  color: var(--rdp-text-color);
+  width: 80%;
+  max-width: fit-content;
+  margin: 0 auto;
+  overflow-wrap: break-word;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
We should have a way to generalize this media query. Maybe with a class or by using a `main` element for the main content